syft/distro/identify.go

```diff
@@ -90,35 +90,21 @@ identifyLoop:
 		}
 	}
 
-	if distro != nil && distro.Type == UnknownDistroType {
```

Comment:
Since we're always taking the supplied field and not returning a strong type I don't know if this check is still neccesary. 

Should we be checking if distro.Type is `""`

Reply:
Good point! It looks like it's no longer that "we failed to identify a thing" — we're just always surfacing what's there, which may be nothing

Reply:
Thinking more on this — I'm even wondering what use we have for a "nillable" value to be returned for all of these related functions (that is, return type of `*Distro` and conditionally returning a `nil` value). What's the difference between that and a zero-value `Distro` struct?